src/main/java/com/google/devtools/build/lib/exec/SpawnStrategyResolver.java

```diff
@@ -86,11 +87,13 @@ public List<? extends SpawnStrategy> resolve(
       if (fallbackStrategies.isEmpty()) {
         String message =
             String.format(
-                "%s spawn cannot be executed with any of the available strategies: %s. Your"
+                "%s spawn%s cannot be executed with any of the available strategies: %s. Your"
                     + " --spawn_strategy, --genrule_strategy and/or --strategy flags are probably"
                     + " too strict. Visit https://github.com/bazelbuild/bazel/issues/7480 for"
                     + " advice",
-                spawn.getMnemonic(), strategies);
+                spawn.getMnemonic(),
+                Spawns.requiresSandboxing(spawn) ? ", which requires sandboxing," : "",
```

Comment:
For your consideration: would it be better to write "which requires sandboxing due to path mapping" here, thus making the reason absolutely clear? (But then we should also rename `requiresSandboxing to `requiresPathMapping`.)

Reply:
I agree that's better at this point, it can always be generalized later.